### ISSUE TITLE
Information about the type of overlaps the other panels. This is very bad.

### ISSUE THREAD

**Issue by mezoni** [dart-atom/dartlang#939]
See picture.

![default](https://cloud.githubusercontent.com/assets/2311549/14503302/f8a16fda-01c8-11e6-8c49-df02884152c7.png)

**Comment by devoncarew** [comment]
Yup, agreed! The UI is provisional - we'll replace it with something that looks more like the F1 help tooltip (and that handles mouse enter/leave and focus changes better).

**Comment by mezoni** [comment]
Problems found.

The concepts of the `tooltip` and` outline` are not properly integrated into the concept of "dart editor".
The are not correlated. This is good and bad at the time.
This is good because they should not to know each other.
This is bad because "concept of ui elements of the dart editor" does not takes into account that they can conflict because they shares the same resource - pointing device (mouse) and hey can overlaps on the screen.

P.S.
Also `StatusDisplay` (as and any other custom editor elements) can sometimes conflict with `tooltip`.
